src/homepageExperience/components/steps/Finish.tsx

```diff
@@ -61,6 +61,10 @@ const fireConfetti = () => {
   })
 }
 
+const handleNextStepEvent = (nextStepName: string) => {
+  event(`firstMile.nextSteps.${nextStepName}.clicked`)
```

Comment:
let's also add the `props.wizardEventName` here, so something like:

```ts
event(`firstMile.${props.wizardEventName}.nextSteps.${nextStepName}.clicked`)
```

Reply:
oops, added now.